x-pack/test/security_solution_api_integration/test_suites/entity_analytics/utils/asset_criticality.ts

```diff
@@ -155,6 +156,18 @@ export const assetCriticalityRouteHelpersFactory = (
       .set(X_ELASTIC_INTERNAL_ORIGIN_REQUEST, 'kibana')
       .expect(expectStatusCode);
   },
+  bulkUpload: async (
+    records: Array<{ id_field: string; id_value: string; criticality_level: string }>,
```

Comment:
shouldn't we import the actual CriticalityRecord type here?

Reply:
Yep!